Content.Tests/DMProject/Tests/Text/Splittext.dm

```diff
@@ -0,0 +1,38 @@
+/proc/list_equal_asert(list/A, list/B)
+	ASSERT(length(A) == length(B))
+	ASSERT(A ~= B)
```

Comment:
This entire helper proc can be removed and replaced with `ASSERT(testX ~= testX_expected)`